src/main/java/org/wso2/lsp4intellij/client/DefaultLanguageClient.java

```diff
@@ -265,4 +290,54 @@ public void logMessage(MessageParams messageParams) {
     protected final ClientContext getContext() {
         return context;
     }
+
+    @Override
+    public CompletableFuture<Void> createProgress(WorkDoneProgressCreateParams params) {
+        String token;
+        if (params.getToken().getLeft() != null) {
+            token = params.getToken().getLeft();
+        } else {
+            token = params.getToken().getRight().toString();
+        }
```

Comment:
Shall we use a `else if` here instead, to avoid potential NPE where both `getLeft()` and `getRight()` return null?
```suggestion
        } else {
            token = params.getToken().getRight().toString();
        }
```